documentation/modules/auxiliary/scanner/mssql/mssql_login.md

```diff
@@ -15,6 +15,41 @@ A docker container can be spun up with the following command to test this module
 
 ## Options
 
+### CreateSession
+
+When using the mssql_login module, the CreateSession option can be used to obtain an interactive
+session within the smb instance. Running the following commands with all other options set:
+
+```
+set CreateSession true
```

Comment:
Thoughts on maybe adding more steps here on using a login module. I.e copy pasta the full steps so if a user wants to try this then don't have to go elsewhere in the docs to figure out how to use the login modules. I think a single example of lets say `mssql_login` would be fine.

Reply:
I think this already exists below in 'Scenarios' - do you think we should restructure things to make it more cohesive?